docsrc/manual/scripting-capabilities.rst

```diff
@@ -1229,9 +1229,74 @@ These are some miscellaneous functions that can be very useful when building cus
 
    :param string message: the message to log
 
-
 .. js:function:: map.error(message)
 
    Logs a message to the Porymap log file with the prefix ``[ERROR]``.
 
    :param string message: the message to log
+
+.. js:function:: map.showMessage(text, informativeText, detailedText)
```

Comment:
These should probably exist under some `util`-like namespace instead of `map`, but I don't know if I care that much.

Reply:
Yeah probably. Same for Settings, and the Overlay functions to some degree. Some of the Tileset functions could be explained as applying to the tilesets for the current map, but some of them apply to all tilesets (e.g. `getMaxPrimaryTilesetTiles`). I just worry about losing the convenience of not having to think about what comes before the function name.